modules/op-overview.adoc

```diff
@@ -0,0 +1,101 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/understanding-openshift-pipelines.adoc
+
+:_content-type: CONCEPT
+[id="op-overview_{context}"]
+= Overview of {pipelines-title}
+
+{pipelines-title} is a comprehensive solution for automating CI/CD processes, such as building, testing, and deploying code, in a cloud-native way.
+
+With {pipelines-shortname}, you can use custom OpenShift (Kubernetes) resources to configure your process as a set of “building blocks” that run in containers and pods on {product-title}.
```

Comment:
OpenShift and other product titles should use the attributes from the common attributes file: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#product-title-and-version

Reply:
I also think that the "(Kubernetes)" can be omitted here as assumed knowledge since this is explained as part of core OCP docs

Reply:
I will remove Kubernetes. The problem in this specific context is that "OpenShift resource" is the description of a resource as opposed to a product name. It's never "OpenSnift Container Platform resource" or "OKD resource" or even "Red Hat OpenShift resource" - it is always "OpenShift resource". I was not able to find a suitable attribute. If there is a suitable attribute please point it out.